Formula/d/dotnet.rb

```diff
@@ -2,10 +2,9 @@ class Dotnet < Formula
   desc ".NET Core"
   homepage "https://dotnet.microsoft.com/"
   # Source-build tag announced at https://github.com/dotnet/source-build/discussions
-  url "https://github.com/dotnet/installer.git",
-      tag:      "v7.0.100-rtm.22521.12",
-      revision: "e12b7af219b96b5e07039ea8e3e268380329d72c"
-  version "7.0.100"
+  url "https://github.com/dotnet/dotnet.git",
+      tag:      "v8.0.0",
```

Comment:
Not sure if we should just switch to the .NET runtime version (`8.0.0`) or keep previous scheme of SDK version (`8.0.100`). Bit easier to handle former as it is the tag on new monorepo.

Reply:
Yeah not sure how they're going to handle SDKs as previously one runtime version would have many SDK versions, but source build would only work on one SDK version. I agree with using the tag at this time.